File: JavaScriptPractice/Leetcode434.js
var countSegments = function(s) {

    let ans=0;
    for (let i = 0; i < s.length; i++) {
        if (s[i]!='' &&(i==0 || s[i-1]==' ')) {
            ans++;
        }
        
    }
    return ans;
    
};

File: CSharpPractice/LeetcodePractice/Leetcode434.cs
public class Solution434 {
    public int CountSegments(string s) {
        int ans=0;
        for (var i = 0; i < s.Length; i++)
        {
            if (s[i]!=' ' &&(s[i-1]==' ' || s[0]!=' '))
            {
                ans++;
            }
        }
        return ans;
        
    }
}